bootcamp-java/Topic1/classes/Users.java

```diff
@@ -0,0 +1,118 @@
+package classes;
+
+public class Users {
+
+	private String name, surname, telephoneNum, creditCardNum, deliverAdd, email, username, password;
+	private String rol = "guest";
+	private boolean mailist = false;

```

Comment:
Exacto, si el usuario se suscribe, queda ese atributo en true y nos ahorramos la creación de una clase.

Date: Tue, 20 Jan 2015 06:51:01 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] ShoppingCart (#4)

In bootcamp-java/Topic1/classes/Users.java:

> @@ -0,0 +1,118 @@
> +package classes;
> +
> +public class Users {
> +
> +	private String name, surname, telephoneNum, creditCardNum, deliverAdd, email, username, password;
> +	private String rol = "guest";
> +	private boolean mailist = false;

La idea es que si tiene mailingList en true el usuario recibe una notificación no? De esta forma no se necesita una clase mailing list. Me parece bien.


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =